### ISSUE TITLE
This version of alpm.rs does not support libalpm v13.0.2 only v14.x.x is supported

### ISSUE THREAD

**Issue by siakc** [Morganamilo/paru#1166]
### Affected Version
Trying to update to paru-git 2.0.3.rebuild.1.r0.g5355012-1 
Pacman v6.0.2 - libalpm v13.0.2

### Description
There were similar issues but think their versions were different. Paru fails to update itself.

### Output

```sh
paru -Syu
... (to long to be included here)...
   Compiling phf v0.11.2
error: failed to run custom build command for `alpm v3.0.4 (https://github.com/archlinux/alpm.rs?rev=306342#306342ef)`

Caused by:
  process didn't exit successfully: `/home/siak/.cache/paru/clone/paru-git/src/paru/target/release/build/alpm-abfddbca03c397bc/build-script-build` (exit status: 101)
  --- stderr
  thread 'main' panicked at /home/siavoshk/.cargo/git/checkouts/alpm.rs-a0070a235cf20bfa/306342e/alpm/build.rs:25:13:
  this version of alpm.rs does not support libalpm v13.0.2 only v14.x.x is supported
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
==> ERROR: A failure occurred in build().
    Aborting...
error: failed to build 'paru-git-2.0.1.r1.g584fb75-1': 
error: packages failed to build: jdupes-1.27.3-2  paru-git-2.0.1.r1.g584fb75-1

```

**Comment by efernau** [comment]
I think I have a related problem.

 `paru: error while loading shared libraries: libalpm.so.13: cannot open shared object file: No such file or directory`

solved with:

`sudo ln -s /usr/lib/libalpm.so.14.0.0 /usr/lib/libalpm.so.13`

**Comment by dscottboggs** [comment]
Getting the same error with `cargo install paru` after updating

**Comment by FantasyPvP** [comment]
> Getting the same error with `cargo install paru` after updating

same issue here. the crates.io version of paru is using an outdated 1.x.x build which is not compatible with the latest software on arch.
the solution is simply for the maintainer to update the crates.io release

**Comment by Cxarli** [comment]
For others that arrived here after upgrading their system, this workaround works for me now:

```
cargo install --git https://github.com/Morganamilo/paru.git
```

**Comment by siakc** [comment]
> For others that arrived here after upgrading their system, this workaround works for me now:
> 
> ```
> cargo install --git https://github.com/Morganamilo/paru.git
> ```

`error: failed to run custom build command for `alpm v3.0.4 (https://github.com/archlinux/alpm.rs?rev=306342#306342ef)`

Caused by:
  process didn't exit successfully: `/tmp/cargo-install0C07mw/release/build/alpm-abfddbca03c397bc/build-script-build` (exit status: 101)
  --- stderr
  thread 'main' panicked at /home/siavoshk/.cargo/git/checkouts/alpm.rs-a0070a235cf20bfa/306342e/alpm/build.rs:25:13:
  this version of alpm.rs does not support libalpm v13.0.2 only v14.x.x is supported
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
`

**Comment by TudbuT** [comment]
Can be fixed by updating alpm to v3.0.5 and using the git version of aur-depends (i had problems with it not cloning manually, you can clone it manually and then refer to it by path).

**Comment by afiorillo** [comment]
I encountered this because my device runs Manjaro rather than stock Archlinux. While the [core pacman](https://archlinux.org/packages/core/x86_64/pacman/) is 6.1.0-3, the [Manjaro stable pacman](https://software.manjaro.org/package/pacman) is 6.0.2-18 which itself provides libalpm 13.

My solution to this was to switch to Manjaro's `testing` branch, which includes a newer version of Pacman. Your mileage may vary; this solution isn't for everyone.

```bash
# get pacman 6.1.0
sudo pacman-mirrors --api --set-branch testing
sudo pacman -S pacman libpamac
# Paru's install instructions, assuming I already have `base-devel`
git clone https://aur.archlinux.org/paru.git
cd paru
makepkg -si
# System update everything else
paru
```
